paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -52,6 +53,7 @@ class Paparazzi(
   private val environment: Environment = detectEnvironment(),
   private val deviceConfig: DeviceConfig = DeviceConfig.NEXUS_5,
   private val theme: String = "android:Theme.Material.NoActionBar.Fullscreen",
+  private val disableAppCompat: Boolean = false,
```

Comment:
would prefer this was in the positive `appCompatEnabled: Boolean = true`

Reply:
updated